src/jti_background.js

```diff
@@ -77,6 +79,30 @@ function getData (callback) {
     });
 }
 
+// Return the "rate it now" flag
+function getRateStatus (callback) {
```

Comment:
I think it would be better to make this, and the setRateStatus general toggle functions. You can specify which toggle you want to get and set. That way we won't need to write other functions if we need to add other toggles down the road.